_modules/conditionals.md

```diff
@@ -6,3 +6,119 @@ short_title: Buttons and Switches
 icon: fa-solid fa-toggle-on
 order: 6
 ---
+
+## What are Conditionals?
+
+In this module, we'll explore **conditionals** which are statements in a program that evaluate to `true` or `false`. You've already seen conditionals when we learned about the **loop condition** in the last module. We'll begin here by exploring the **if statement**, which looks like the following:
+
+```c
+if (buttons_get(1)) {
+    leds_set_color(1, 255, 0, 0);
+}
+```
+
+This block of code tells our board to turn on LED1 _if_ Button1 is pressed. The function `buttons_get()` can be used as a **conditional** because it returns a `true` or `false` value that indicates whether the button that was passed into it (in this case Button1) is pressed. Notice that in an **if statement** the **conditional** is placed in parenthesis and the commands to execute if it is true are placed in curly braces.
+
```

Comment:
I think you should introduce && and || here.

Reply:
Done